src/templates/snek/gql/errors.ts

```diff
@@ -0,0 +1,57 @@
+//#region > Imports
+//> Sessions
+// Contains the SNEK session
+import { SnekSession } from "../../../session/sessions";
+//> Types
+// Contains the type declaration for apollo results
+import { ApolloResult } from "./index";
+//#endregion
+
+//#region > Classes
+/** @class A set of session aware tasks */
+class TaskError {
+  /**
+   * @constructor
+   * @author Nico Schett <contact@schett.net>
+   * @param {SnekSession} session A session for the task
+   */
```

Comment:
Should have some sort of comment.

Initializes session aware Tasks.

Reply:
Yeah, but not your suggestion cause this includes too less/useless information.
We have to discuss about a new TONIC rule in snek-at/tonic/issues/3